test/GCongr/inequalities.lean

```diff
@@ -169,6 +170,12 @@ example {a b c d e : ℝ} (_h1 : 0 ≤ b) (_h2 : 0 ≤ c) (hac : a * b + 1 ≤ c
   guard_target =ₛ a * b ≤ c * d
   linarith
 
+-- test big operators
+example (f g : ℕ → ℕ) (s : Finset ℕ) (h : ∀ i ∈ s, f i ≤ g i) :
```

Comment:
Do f, g and s need to be explicit? (I am aware the same question applies to the next example also.)

Reply:
No, that is irrelevant

Reply:
:+1: In this case: do you mind switching them to implicit? (If this were a mathlib file, this is what I'd ask a PR author to do.)

Reply:
I don't think this really matters for test files. This is not something we're careful about at all (see for instance, the rest of the this file).

Reply:
Sure, I'm happy to adjust to that policy as well.